kiwi/utils/sysconfig.py

```diff
@@ -36,7 +36,7 @@ def __setitem__(self, key, value):
         self.data_dict[key] = value
 
     def __getitem__(self, key):
-        return self.data_dict[key]
+        return self.data_dict.get(key)
```

Comment:
I really doubt it can create any issue, however this looks misleading to me. If I run something like `sysconfig_file['nonexistingKey']` I'd expect it to raise a KeyError Exception, however with this code it returns `None`. I think it is good to reuse the dictionary `__getitiem__` here, this way we get an analog behavior.

Reply:
ok, I'm gonna change things to be consistent with standard use of dicts